source/js/classes/crop-host.js

```diff
@@ -379,22 +398,20 @@ crop.factory('cropHost', ['$document', 'cropAreaCircle', 'cropAreaSquare', 'crop
         theArea = new CropAreaCircle(ctx, events);
 
         // Init Mouse Event Listeners
-        $document.on('mousemove', onMouseMove);
+        elCanvas.on('mousemove', onMousehover);
```

Comment:
if onMousehover is purely for styling I would suggest a slight name change to onMouseOver as it's slightly more obvious what its doing then. Probably don't need any of the touch logic inside it as it's not applicable